web/pdf_viewer.js

```diff
@@ -1099,10 +1114,24 @@ class PDFViewer {
       newScale * PixelsPerInch.PDF_TO_CSS_UNITS
     );
 
-    const updateArgs = { scale: newScale };
+    const updateArgs = {
+      scale: newScale,
+      postponeDrawingDelay: delay,
```

Comment:
We should really only pass along this value if it's *actually valid*, please also see the comment below.